source/settings/telemetry.txt

```diff
@@ -0,0 +1,132 @@
+.. _compass-telemetry:
+
+===========================
+Configure Telemetry Options
+===========================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+|compass-short| collects anonymous aggregated usage data to improve
+MongoDB products. |compass-short| collects this information by default,
+but you can disable this data collection.
+
+About this Task
+---------------
+
+|compass-short| tracks the following data:
+
+- The type of MongoDB deployment |compass-short| is connected to. For
+  example, Enterprise Edition, Community Edition, or Atlas Data Lake.
+
+- User actions performed in |compass-short|, such as creating
+  collections and indexes. |compass-short| only tracks the general
+  action performed, and does not track field values or namespaces.
+
+- Errors.
+
+|compass-short| *does not* track:
+
+- IP addresses, hostnames, usernames, or credentials.
+
+- Queries run in |compass-short|.
+
+- Data stored in your MongoDB deployment.
+
+- Personal identifiable information.
```

Comment:
Since all of these list items are fragments, I think we can leave out the periods re [the style guideline on periods ](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#periods)

Reply:
Fixed